projects/steering_digital_input/inc/digital_input_events.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+//Events IDs for digital input
+typedef enum {
+  STEERING_DIGITAL_INPUT_EVENT_HORN = 0,
+  STEERING_DIGITAL_INPUT_EVENT_RADIO_PPT,
+  STEERING_DIGITAL_INPUT_EVENT_HIGH_BEAM_FWD,
+  STEERING_DIGITAL_INPUT_EVENT_HIGH_BEAM_BACK,
+  STEERING_DIGITAL_INPUT_EVENT_REGEN_BRAKE_TOGGLE,
+  STEERING_DIGITAL_INPUT_EVENT_HEADLIGHTS_ON_OFF,

```

Comment:
probably not gpio. 
Also, if you're having the `on/off` capability in the event's data, then don't  name this <EVENT>_ON_OFF, just name it <EVENT>, and have the data be ON or OFF